src/routers/blog.js

```diff
@@ -0,0 +1,79 @@
+const { Router } = require('express');
+const util = require('../util');
+const logger = require('../logger');
+const router = new Router();
+
+const fs = require('fs');
+const path = require('path');
+const marked = require('marked');
+const matter = require('gray-matter');
+
+router.get('/', async (request, response) => {
+
+	const reqLocale = request.locale;
+	const locale = util.getLocale(reqLocale.region, reqLocale.language);
+
+	const localeString = reqLocale.toString();
+
+	const fileList = fs.readdirSync('blogposts');
+
+	// We get the info for each blogpost, ignoring the ones starting with _
+	const postList = fileList
+		.filter(filename => !filename.startsWith('_'))
+		.filter(filename => filename.endsWith('.md')) //Ignores other files/folders
```

Comment:
This is a really small thing but can you do me a favor and put a space after the `//`